### DISCUSSION TITLE
Don't understand use of '---' for relation

### DISCUSSION THREAD

**Discussion by ouhouhsami** [vzhd1701/csv2notion#19] [General]
Hi there, 

thx for your library :) 

I have a problem trying --column-types '---' to force the library make (use a) relation.

I get a 'Unknown types: relation; allowed types: person, checkbox, rollup, last_edited_by, created_time, number, formula, text, phone_number, email, date, status, created_by, select, file, last_edited_time, multi_select, url' error message. 
The '---' type, is not available contrary to what we can read, if I understand, in the documentation.

Have you got an idea to help me deal with relations? (a working example with csv and command line options for instance).

Again, thanks a lot.

**Comment by vzhd1701** [comment]
csv2notion cannot create new relation columns, it can only use existing ones that you set up manually. That's why there is a '---' in that table. Sorry for confusion.
